packages/gdl-frontend/components/BookDetailsPage/Metadata.js

```diff
@@ -22,19 +23,32 @@ type Props = {
   online: boolean
 };
 
-const Contributor = ({ contributorType, values }) => (
-  <>
-    <Typography variant="subtitle2" component="span">
-      <Plural
-        one={contributorType}
-        other={`${contributorType}s`}
-        value={values.length}
-      />
-    </Typography>
-    <Typography component="span" paragraph css={noMarginForLastChild}>
-      {values.map(contributor => contributor.name).join(', ')}
-    </Typography>
-  </>
+type ContributorProps = {
+  contributorType: string,
+  values: Array<{ name: string }>,
+  intl: intlShape
+};
+
+const contributorMessages = defineMessages({
+  contributor: {
+    id: 'Contributor',
+    defaultMessage: '{contributorType}'
+  }
+});
+
+const Contributor = injectIntl(
+  ({ contributorType, values, intl }: ContributorProps) => (
+    <>
+      <Typography variant="subtitle2" component="span">
+        {intl.formatMessage(contributorMessages.contributor, {
+          contributorType: `${contributorType}${values.length > 1 ? 's' : ''}`
```

Comment:
Er det slik at denne sjekken tar bare hensyn til flertall og enkeltform for engelsk `contributor | contributors`?

Reply:
Mulig vi skal ta å ha to forskjellige oversettelser her for å få det så enkelt som mulig. Så da retter vi på dette etter at vi har bare brukt `id`.